### ISSUE TITLE
SOMweb Garagenmotorsteuerung

### ISSUE THREAD

**Issue by Patrickoli** [ioBroker/AdapterRequests#820]
**What kind of device or service would you like to see an adapter for?**
I have a garage motor from the company SOMMER Antriebs- und Funktechnik GmbH in use. To control the garage door with a smartphone, you need a device/hub/bridge (SOMweb), which is the connection between the garage motor and the network. SOMweb allows controlling drives via a smartphone app or a web browser.
Unfortunately, I am not aware of an official interface.
However, I found the following Python project:
[https://pypi.org/project/somweb/](url)

Manufacturer/device link
[https://www.sommer.eu/de/somweb.html](url)

Instruction manual
[https://downloads.sommer.eu/files/SOMweb_S11717-00000.pdf](url)

**Is the device connected to the internet or only available on a local network?**
The device is accessible both via the Internet and locally.

**Is an official App or Website available?**
Manufacturer/Device Link
[https://www.sommer.eu/de/somweb.html](url)

Instruction manual
[https://downloads.sommer.eu/files/SOMweb_S11717-00000.pdf](url)

**Is an official API including documentation available?**
Unfortunately, an official interface is not known to me.
However, I have found the following Python project:
[https://pypi.org/project/somweb/](url)

**Are other libraries for an integration available?**
Unfortunately, an official interface is not known to me.
However, I have found the following Python project:
[https://pypi.org/project/somweb/](url)

**Is this device already integrated in other Smart Home systems?**
The SOMweb supports voice control via Siri (HomeKit) and Google Assistant. In addition, the SOMweb can also be controlled via IFTTT with Amazon Alexa.

**Is this device already integrated in homebridge? Might the ham adapter in combination with the homebridge plugin be sufficient?**
No idea

**Additional context**
You can also generate the device and data points with the HomeKit controller in ioBroker, but I subsequently failed to integrate the garage door with HomeKit using Yahka.

**Comment by Apollon77** [comment]
In fact no need to switch .. you can also use the "ioBroker.hass" Adapter to just use Home assistent for what you need from there :-)

**Comment by Patrickoli** [comment]
Yes, but I don't run two systems because of the one data point. :-)Am 21.10.2024 um 16:23 schrieb Ingo Fischer ***@***.***>:﻿
In fact no need to switch .. you can also use the "ioBroker.hass" Adapter to just use Home assistent for what you need from there :-)
